dali/operators/math/normalize/normalize.cc

```diff
@@ -142,10 +147,19 @@ void Normalize<CPUBackend>::FoldStdDev() {
   if (v == 0) {
     return;
   }
-  float rdiv = static_cast<float>(1.0 / v);
+  float rdiv = 0;
+  float scale = scale_;
+  if (v > degrees_of_freedom_) {
+    rdiv = static_cast<float>(1.0 / (v - degrees_of_freedom_));
+  } else {
```

Comment:
Shouldn't we error in such cases?

Reply:
Numpy will give you infinite stddev - so dividing by it will produce zero - I'm sort-of emulating that.